resources/public/cheatSheet.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset='utf-8'>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Metafacture Playground Cheat Sheet</title>
+    <link rel="icon" href="images/metafacture-logo-removebg.png">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.7/semantic.min.css"></link>
+  </head>
+  <body>
+	<h1>Metafacture Playground Cheat Sheet </h1>
+Here are some examples of workflows. They will open in playground.
+Search for something of your interest. You can combine those workflows by copy'n'paste - while not everything is free combinable in the 'Flux' (e.g. some input formats depend on special treatments) you often can combine a section with another.
+The 'Fix' is mostly exchangeable, but sometimes a serializer (the section following the 'fix' command) demands some fields as inputs. </br>
+</br>
+Just try it.</br>
+If you have a task you cannot resolve by yourself, let us know - we will provide a solution here!
+</br></br>
+Don't forget to click "process" in the playground to get a result.
+	<h2>
+	Analyzing
+	</h2>
+	<a href="https://metafacture.org/playground/?flux=%22https%3A//d-nb.info/1106253078/about/marcxml%22%0A%7C+open-http%28accept%3D%22application/xml%22%29%0A%7C+decode-xml%0A%7C+handle-marcxml%0A%7Cfix%28%22nothing%28%29%22%2CrepeatedFieldsToEntities%3D%22true%22%29%0A%7Cflatten%0A%7Cencode-literals%0A%7Cprint%0A%3B&active-editor=fix">List all MarcXML fields - gets all field names (and their values)</a>
+&nbsp;&nbsp;;&nbsp;&nbsp;
```

Comment:
Looking through these examples. We could add them as usecase examples.
https://metafacture.org/playground/?flux=%22https%3A//d-nb.info/1106253078/about/marcxml%22%0A%7C+open-http%28accept%3D%22application/xml%22%29%0A%7C+decode-xml%0A%7C+handle-marcxml%0A%7Cfix%28%22nothing%28%29%22%2CrepeatedFieldsToEntities%3D%22true%22%29%0A%7Cflatten%0A%7Cencode-literals%0A%7Cprint%0A%3B